scripts/github-actions/update-browser-versions.js

```diff
@@ -10,33 +9,16 @@ const CHROME_BETA_KEY = 'chrome-beta-version'
 const CIRCLECI_WORKFLOWS_FILEPATH = path.join(__dirname, '../../.circleci/src/pipeline/@pipeline.yml')
 
 // https://developer.chrome.com/docs/versionhistory/reference/#platform-identifiers
-const getLatestVersionData = ({ channel, currentVersion }) => {
-  const options = {
-    hostname: 'versionhistory.googleapis.com',
-    port: 443,
-    path: `/v1/chrome/platforms/linux/channels/${channel}/versions?filter=version>${currentVersion}&order_by=version%20desc`,
-    method: 'GET',
-  }
-
-  return new Promise((resolve, reject) => {
-    const req = https.request(options, (res) => {
-      let response = ''
-
-      res.on('data', (d) => {
-        response += d.toString()
-      })
+const getLatestVersionData = async ({ channel, currentVersion }) => {
+  const url = `https://versionhistory.googleapis.com/v1/chrome/platforms/linux/channels/${channel}/versions?filter=version>${currentVersion}&order_by=version%20desc`
```

Comment:
does this need to be encoded?
```suggestion
  const url = `https://versionhistory.googleapis.com/v1/chrome/platforms/linux/channels/${channel}/versions?filter=version>${currentVersion}&order_by=version desc`
```

Reply:
Maybe not in this instance, but it is safer to encode as that is how strings should be formatted for fetch.

Reply:
I think for myself it was more of a consistency thing where either its entirely decoded or encoded. I'd like decoded and passed to an encoding before calling the fetch but honestly this is just bike shedding at this point and is totally fine 😂